### PR TITLE
Old template positions

### LINKED ISSUE #2044: Some old default positions are missing in the new theme
## Type

- Enhancement

## Problem description

Some core modules expect them to be there like the login widget or the search box

## Steps to reproduce

Install fork 5 and look in the backend at the pages

## Expected behavior / Proposed solutions

I think it would be best to add the missing position(s) back

### PR DESCRIPTION
**Pull request by jonasdekeukelaere**
## Type
- Enhancement

## Resolves the following issues
Fixes #2044 

## Pull request description
Will bring back the old template positions.

### CODE REVIEW

**Comment by jonasdekeukelaere** [comment]
Preview: 
screen shot 2017-06-08 at 14 42 03

**Comment by jonasdekeukelaere** [comment]
WIP: still need to (re-)add the homepage positions "left" and "right".

**Comment by jonasdekeukelaere** [comment]
Note to self: positions also needs to be set in (extensions) installer.

**Comment by jonasdekeukelaere** [comment]
@carakas the MR about fork new theme has already been merged in this branch, fyi

File: src/Backend/Modules/Pages/Installer/Installer.php [hunk 1]
```diff
@@ -90,7 +90,7 @@ private function installExampleData(): void
                     ['html' => __DIR__ . '/Data/' . $language . '/sample1.txt'],
                     ['extra_id' => $extras['blog_widget_recent_articles_list'], 'position' => 'main'],
                     ['extra_id' => $extras['blog_widget_recent_comments'], 'position' => 'main'],
-                    ['extra_id' => $extras['search_form'], 'position' => 'top']
+                    ['extra_id' => $this->getExtraId('sitemap'), 'position' => 'top']
```
Comment (carakas) [inline src/Backend/Modules/Pages/Installer/Installer.php]: This shouldn't be sitemap but it should be the search_form, it is a bug that has slipped in when the installer was updated

File: src/Backend/Modules/Pages/Installer/Installer.php [hunk 2]
```diff
@@ -107,7 +107,7 @@ private function installExampleData(): void
                     ['extra_id' => $extras['blog_widget_categories'], 'position' => 'main'],
                     ['extra_id' => $extras['blog_widget_archive'], 'position' => 'main'],
                     ['extra_id' => $extras['blog_widget_recent_articles_list'], 'position' => 'main'],
-                    ['extra_id' => $extras['search_form'], 'position' => 'top']
+                    ['extra_id' => $this->getExtraId('sitemap'), 'position' => 'top']
```
Comment (carakas) [inline src/Backend/Modules/Pages/Installer/Installer.php]: This shouldn't be sitemap but it should be the search_form, it is a bug that has slipped in when the installer was updated

File: src/Backend/Modules/Pages/Installer/Installer.php (hunk 1 above)
Comment (jonasdekeukelaere) [inline src/Backend/Modules/Pages/Installer/Installer.php]: Must be a mistake by @jeroendesloovere - fixed 50c4bf5

File: src/Backend/Modules/Pages/Installer/Installer.php (hunk 2 above)
Comment (jonasdekeukelaere) [inline src/Backend/Modules/Pages/Installer/Installer.php]: Fixed 50c4bf5

**Pull request merged**